include/defines.h

```diff
@@ -70,6 +70,25 @@
 #define PEACH  6
 #define BOWSER 7
 
+/**
+ * Options for gMenuSelection
+**/
+#define START_MENU_DUPLICATE 0
+#define GAME_SELECT_MENU_DUPLICATE 1
+#define PLAYER_SELECT_MENU_DUPLICATE 2
+#define COURSE_SELECT_MENU_DUPLICATE 3
```

Comment:
I ended up swapping these lower values to the 'duplicates' since the values below are the ones that are actually being set in the code.